### PR TITLE
Update Schedule & refactor times mentioning in each doc page

### PR DESCRIPTION
**Pull request by ericholscher**
This is a quick refactor that moves NA schedule info into the main schedule,
and updates the schedule a bit.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -125,6 +125,8 @@
 
 # html_context.update(main())
 
+# html_experimental_html5_writer = True
```
Comment (plaindocs) [inline docs/conf.py:128]: :+1:

File: docs/conf/portland/2018/schedule.rst [hunk 2]
```diff
@@ -18,54 +20,55 @@ Saturday, May 5
...
-    :backlinks: none
-
 Writing Day
 ~~~~~~~~~~~
 
-* **Where**: Main stage, Crystal Ballroom
+Get together with other documentarians and work on an open source project and learn some new skills.
+
+* **Where**: Crystal Ballroom
 * **When**: **9am-5pm**
```
Comment (plaindocs) [inline docs/conf/portland/2018/schedule.rst:39]: Gonna have to address the EU / US time styles at some point :/

File: docs/conf/portland/2018/schedule.rst [hunk 3]
```diff
@@ -18,54 +20,55 @@ Saturday, May 5
...
 -------------
 
-.. contents::
-    :local:
-    :backlinks: none
+Unconference
+~~~~~~~~~~~~
+
+Have in depth discussions about a large number of topics.
+This is scheduled in real time, so feel free to suggest your own session!
```
Comment (plaindocs) [inline docs/conf/portland/2018/schedule.rst:59]: Maybe "on the day" instead of "in real time" ?

File: docs/conf/portland/2018/tickets.rst [hunk 4]
```diff
@@ -4,59 +4,82 @@
 Tickets
 =======
 
-**Tickets on sale now!**
+{% if flagticketsonsale %}
+
+**Tickets are on sale now!**
+
+{% else %}
```
Comment (plaindocs) [inline docs/conf/portland/2018/tickets.rst:11]: We should probably add a `flagticketssoldout` so this doesn't break when you generate a new conf.

File: docs/conf/portland/2018/tickets.rst [hunk 5]
```diff
@@ -4,59 +4,82 @@
...
 * Wifi throughout the event
-* Meeting lots of fantastic people in a spacious, inviting venue.
-
+* Meeting lots of fantastic people in a spacious, inviting venue
 
 .. class:: ticket
 
-Community Sponsorship Tickets *$750*
-------------------------------------
+**Community Sponsorship Tickets** *{{tickets.community.price}}*
```
Comment (plaindocs) [inline docs/conf/portland/2018/tickets.rst]: :+1:

**Review by plaindocs — COMMENTED** [review]
Nice bit of cleanup and refactoring <3

**Comment by ericholscher** [comment]
Merged this into #474, closing this.

**Pull request closed without merging**